src/main/java/htsjdk/samtools/metrics/MetricsFile.java

```diff
@@ -48,17 +48,19 @@
  *
  * @author Tim Fennell
  */
-public class MetricsFile<BEAN extends MetricBase, HKEY extends Comparable> implements Serializable {
+public class MetricsFile<BEAN extends MetricBase, HKEY extends Comparable<HKEY>> implements Serializable {
+    private static final long serialVersionUID = 1L;
+
```

Comment:
I'm not aware of a case where we've needed to serialize a MetricsFile, but we should probably add a test to make sure it works..